packages/arb-token-bridge-ui/src/hooks/useNetworkTPS.ts

```diff
@@ -0,0 +1,55 @@
+// Used to fetch the TPS (Transactions per Second) information for a network
+
+import axios from 'axios'
+import { load } from 'cheerio'
+import { getExplorerUrl, isNetwork } from 'src/util/networks'
+import useSWR from 'swr'
+import { useNetworksAndSigners } from './useNetworksAndSigners'
```

Comment:
nit: can add a new line to separate relative imports

Reply:
Done